### PR TITLE
New Command for listing environments

### PR DESCRIPTION
**Pull request by JGiola**
## What this PR is for?

This PR introduce a new first level command `runtime` and its first subcommand `environment list` that will show the user all runtime environment associated to a given project.

### CODE REVIEW

File: internal/cmd/environments/environments.go [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+)
+
+func EnvironmentCmd(o *clioptions.CLIOptions) *cobra.Command {
+	runtimeCmd := &cobra.Command{
+		Use:   "environment",
+		Short: "Manage resources deployed with Mia-Platform Console",
+		Long: `Manage resources deployed with Mia-Platform Console.
+
+Once a project from Mia-Platform Console is deployed at least once, you can manage and monitor
+the resources generated, like Pods, Cronjobs and logs.
```
Comment (davidebianchi) [inline internal/cmd/environments/environments.go:43]: We should add some environment description. This is the same of the runtime

File: internal/cmd/environments/environments.go [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+	"github.com/spf13/cobra"
+)
+
+const (
+	listEnvironmentsEndpointTemplate = "/api/backend/projects/%s"
+	getClusterEndpointTemplate       = "/api/tenants/%s/clusters/%s"
+)
+
+func EnvironmentCmd(o *clioptions.CLIOptions) *cobra.Command {
+	runtimeCmd := &cobra.Command{
```
Comment (davidebianchi) [inline internal/cmd/environments/environments.go:37]:
```suggestion
	environmentCmd := &cobra.Command{
```

File: internal/cmd/environments/environments.go [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+the resources generated, like Pods, Cronjobs and logs.
+`,
+	}
+
+	// add sub commands
+	runtimeCmd.AddCommand(
+		listEnvironmentsCmd(o),
+	)
+
+	return runtimeCmd
```
Comment (davidebianchi) [inline internal/cmd/environments/environments.go:52]:
```suggestion
	environmentCmd.AddCommand(
		listEnvironmentsCmd(o),
	)

	return environmentCmd
```

Comment (JGiola) [reply internal/cmd/environments/environments.go:52]: Ups, copypasta leftovers, I'm fixing them

**Review by davidebianchi — APPROVED** [review]

**Pull request merged**